docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md

```diff
@@ -49,9 +49,9 @@ spring.shardingsphere.rules.sharding.broadcast-tables[0]= # Broadcast tables
 spring.shardingsphere.rules.sharding.broadcast-tables[1]= # Broadcast tables
 spring.shardingsphere.rules.sharding.broadcast-tables[x]= # Broadcast tables
 
-spring.shardingsphere.sharding.default-database-strategy.xxx= # Default strategy for database sharding
-spring.shardingsphere.sharding.default-table-strategy.xxx= # Default strategy for table sharding
-spring.shardingsphere.sharding.default-key-generate-strategy.xxx= # Default Key generator strategy
+spring.shardingsphere.rules.sharding.default-database-strategy.xxx= # Default strategy for database sharding
+spring.shardingsphere.rules.sharding.default-table-strategy.xxx= # Default strategy for table sharding
+spring.shardingsphere.rules.sharding.default-key-generate-strategy.xxx= # Default Key generator strategy
```

Comment:
This is the change history, is it need to modify?

Reply:
Yes, it's used in '/examples' like [Line 34](https://github.com/apache/shardingsphere/blob/c89217636d579a050d59a95e0e532980d5488dd3/examples/shardingsphere-jdbc-example/single-feature-example/extension-example/custom-sharding-algortihm-example/spi-based-sharding-algorithm-example/spi-based-sharding-spring-boot-jpa-example/src/main/resources/application-sharding-databases-tables.properties)

Reply:
What I mean is the doc is for history version, not for current version

Reply:
I got it, it is not necessary to modify